controllers/MailController.php

```diff
@@ -310,7 +310,7 @@ public function actionCreate($userGuid = null)
         // Preselect user if userGuid is given
         if ($userGuid) {
             /* @var User $user */
-            $user = User::find()->where(['guid' => $userGuid])->active()->visible()->filterBlockedUsers()->one();
+            $user = User::find()->where(['guid' => $userGuid])->available()->one();
```

Comment:
@yurabakhtin atm `available()` only covers `visible() && filterBlockedUsers()` and not `active()` should we keep active in place?

Reply:
@luke- The method `visible()` already keeps the `active()` inside, the code is like this:

```php
    public function visible()
    {
        ...
        return $this->active()
            ->andWhere(['IN', 'user.visibility', $allowedVisibilities]);
    }
```